recipes/trawler/build.sh

```diff
@@ -2,15 +2,21 @@
 
 outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 
+# See https://github.com/Ramialison-Lab-ARMI/Trawler-2.0/issues/1
+# Fix space in some hash-bang lines,
+sed -i.bak 's|#! usr/bin/perl|#!/usr/bin/env perl|' bin/*.pl
+# Fix hash-bang lines not to use system Perl,
+sed -i.bak 's|#!/usr/bin/perl|#!/usr/bin/env perl|' bin/*.pl
+rm -rf bin/*.bak
+
 rm -rf test_data examples
 mkdir -p $outdir
 cp -r * $outdir
 
 # Short wrapper script
 cat > $PREFIX/bin/trawler <<EOF
 #!/bin/bash
-export PERL5LIB=$outdir:$outdir/treg_comparator
-$outdir/bin/trawler.pl \$@
-unset PERL5LIB
+# Last line of script to ensure any failure return code is passed on:
+PERL5LIB=$outdir:$outdir/treg_comparator $outdir/bin/trawler.pl \$@
```

Comment:
Good call on that change! You could even add two more enhancements with
```bash
PERL5LIB=$outdir:$outdir/treg_comparator exec $outdir/bin/trawler.pl "\$@"
```
* `exec`: replace caller process with child, see, e.g., https://unix.stackexchange.com/questions/279024/reason-for-exec-in-wrapper-scripts
* `"$@"`: avoid splitting of input parameters, e.g., `trawler "1 2" 3` will call `trawler.pl` with `["1 2", "3"]` instead of `["1", "2", "3"]`